client/components/accordion/docs/example.jsx

```diff
@@ -11,7 +11,7 @@ var Accordion = require( 'components/accordion' ),
 	Gridicon = require( 'components/gridicon' );
 
 module.exports = React.createClass( {
-	displayName: 'Accordions',
+	displayName: 'Accordion',
```

Comment:
Right, I should renamed to `AccordionExample` instead of. I'll change it.

Side Note I wouldn't implement all changes in the same PR since it's increasing too much.
After this we should walk for each example, step by step, fixing/improving the code.
For now the wrapper component has the enough _intelligence_ to handle names ending with and without `Example` string.